ic-stable-structures/src/storage/structures.rs

```diff
@@ -0,0 +1,106 @@
+use std::collections::{hash_map::Entry, HashMap};
+
+use ic_exports::candid::Principal;
+use ic_exports::ic_kit::ic;
+use ic_exports::stable_structures::{btreemap, cell, memory_manager::MemoryId, Storable};
+
+use super::{error::Error, Memory};
+
+/// Stores value in stable memory, providing `get()/set()` API.
+pub struct StableCell<T: Storable> {
+    data: HashMap<Principal, cell::Cell<T, Memory>>,
+    default_value: T,
+    memory_id: MemoryId,
+}
+
+impl<T: Storable> StableCell<T> {
+    /// Create new storage for values with `T` type.
+    pub fn new(memory_id: MemoryId, default_value: T) -> Self {
+        Self {
+            data: HashMap::default(),
+            default_value,
+            memory_id,
+        }
+    }
+
+    /// Returns reference to value stored in stable memory.
+    pub fn get(&self) -> &T {
```

Comment:
Shouldn't this return an `Option`?

Reply:
The `StableCell` is always has a value. It initialized with default value and updates with `set` method. Also, we can't remove value from `StableCell`. So, no need in `Option`.